config.json

```diff
@@ -80,6 +80,16 @@
 			"name" : "Use alternate status icons",
 			"type" : "checkbox"
 		},
+		{
+			"key" : "all-status-types",
+			"name" : "Remember the selection of 'all status types' on the record status dashboard",
+			"type" : "checkbox"
+		},
+		{
+			"key" : "show-role-name",
+			"name" : "Show the user's role below their username",
+			"type" : "checkbox"
+		},
 		{
 			"key" : "alerts-simplified-view",
 			"name" : "Add a 'simplified view' option to the alerts",
```

Comment:
As the top suggestion to put these checkboxes together but not essential

Reply:
See earlier comment regarding this.